app/js/components/Login.jsx

```diff
@@ -1,52 +1,53 @@
-/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+/** This Source Code Form is subject to the terms of the Mozilla Public License,
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
  *
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-import React from 'react';
-import { hashHistory } from 'react-router';
+import React from "react";
+import { hashHistory } from "react-router";
 
-import LoginForm from './presentational/LoginForm'
+import LoginForm from "./presentational/LoginForm";
```

Comment:
please refactor based on comment above

Reply:
This change will be implemented in the next ticket that reworks the login page: https://issues.openmrs.org/browse/PLAT-42.